runners/java-fn-execution/src/main/java/org/apache/beam/runners/fnexecution/state/StateRequestHandler.java

```diff
@@ -34,6 +35,12 @@
    * <p>Throwing an error during handling will complete the handler result {@link CompletionStage}
    * exceptionally.
    */
-  CompletionStage<BeamFnApi.StateResponse.Builder> handle(BeamFnApi.StateRequest request)
-      throws Exception;
+  CompletionStage<StateResponse.Builder> handle(StateRequest request) throws Exception;
+
+  static StateRequestHandler unsupported() {
```

Comment:
Why package-private? This seems useful anywhere we don't/can't support state.

Reply:
classes inside interfaces are public and not package private.

Drop `static` as it is also a redundant modifier.

Reply:
Ah, I hadn't noticed this was an interface.

I don't think `static` is redundant though.

Reply:
`static` is also redundant.

Reply:
It is definitely not redundant - you must choose between `static`, `default`, or no-implementation for interface methods

Reply:
For some reason I thought it was a class.